review/views.py

```diff
@@ -1,3 +1,110 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from .models import Review
+from user.models import User
+from product.models import Product
+
+class ReviewView(View):
+    def post(self, request):
+        data=json.loads(request.body)
+        print("------request.body", request.body)
+        try:
+            user             = User.objects.get(pk=data['user_id'])
+            product          = Product.objects.get(pk=data["product_id"])
+            content          = data["content"].strip()
+            image_url        = data.get("image_url")
+            user_information = data.get("user_information")
+            star             = data["star"]
+            print("-----user_information---",user_information)
+
+            Review.objects.create(
+                user             = user,
+                product          = product,
+                content          = content,
+                image_url        = image_url,
+                star             = star,
+                user_information = user_information
+            )
+            return JsonResponse({"message": "Review Success!!"}, status=200)
+        except User.DoesNotExist:
+            return JsonResponse({"message":"This nick_name is not exist!"}, status=400)
```

Comment:
```suggestion
            return JsonResponse({"message":"INVALID_USER"}, status=400)
```
이렇게 수정해주세요. 메세지의 코드는 대문자에 띄어쓰기는 _ 로 구분해주세요.

Reply:
넵